bindings/gumjs/gumscript-runtime-java.js

```diff
@@ -2894,7 +2896,9 @@
         const temporaryApi = {
             addLocalReference: null
         };
-        const pending = Process.findModuleByName('libart.so') !== null ? [{
+        temporaryApi.flavor = Process.findModuleByName('libart.so') !== null? 'art' : 'dalvik';
```

Comment:
Perhaps this should be merged with the literal right above it? There should be a space on the left side of the `?` btw.